pkg/policy/types/update.go

```diff
@@ -14,6 +14,15 @@ import (
 
 // PolicyUpdate is a proposed change to a policy in the PolicyRepository.
 type PolicyUpdate struct {
+	// BumpAll, if set to true, forces a policy regeneration for all endpoints
+	// as if policy had changed, without actually changing the contents of the policy repository.
+	//
+	// This is useful when some dependent resources (i.e. Envoy configs) have changed,
+	// so policy must be recomputed.
+	//
+	// If this is true, Rules must be nil. The repository is unchanged.
+	BumpAll bool
```

Comment:
Given there's already another `bool` in here: would it make sense to have some enum-like `type` field instead? That would also prevent encoding invalid state like `BumpAll && ReplaceByLabels`.